cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -589,7 +589,17 @@
   public static final class Notification {
     public static final String TRANSPORT_SYSTEM = "notification.transport.system";
 
-    public static final String KAFKA_DELEGATE_NOTIFICATION_SERVICE = "kafka.delegate.notification.service";
+    /**
+     * Notifications in Streams constants.
+     */
+    public static final class Stream {
+      public static final String STREAM_HEARTBEAT_FEED_CATEGORY = "streamHeartbeats";
+      public static final String STREAM_HEARTBEAT_FEED_NAME = "streamHeartbeats";
```

Comment:
seems like repetitive to me. If the category and feed name are hierarchal, should the category be `stream.internal` and feed name as `heartbeat`? While for "external" feed, the category would be just `stream` and the feed name can be something `ingested.size`?